unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -69,8 +73,8 @@ public UTF8String set(final String str) {
   /**
    * Updates the UTF8String with byte[], which should be encoded in UTF-8.
    */
-  public UTF8String set(final byte[] bytes) {
-    this.bytes = bytes;
+  protected UTF8String set(final byte[] bytes) {
+    this.bytes = (bytes != null) ? bytes : new byte[0];
```

Comment:
Will it be better if we throw exception if null passed in? Or leave it as it was.